hack/update-deps.sh

```diff
@@ -22,10 +22,10 @@ SERVING_VERSION="release-v1.0.1"
 # The list of dependencies that we track at HEAD and periodically
 # float forward in this repository.
 FLOATING_DEPS=(
-  "knative.dev/hack@${KN_VERSION}"
+
   "knative.dev/networking@${KN_VERSION}"
   "knative.dev/operator@${KN_VERSION}"
-  "knative.dev/pkg@${KN_VERSION}"
+
```

Comment:
Did you mean to commit this?

Reply:
I restored in the above commit `restore deps.sh` it is not in?

Reply:
I might've screwed something up on my end in Github, sorry for the noise.

Reply:
np ;)